### PR TITLE
Put out benchmark fires

### PR DESCRIPTION
**Pull request by kazimuth**
Attempting to address #401 #402 #403

### CODE REVIEW

File: crates/bench/benches/generic.rs [hunk 1]
```diff
@@ -282,21 +244,19 @@ fn iterate<DB: BenchDatabase, T: BenchTable + RandomTable>(
...
     table_id: &DB::TableId,
     index_strategy: &IndexStrategy,
     column_index: u32,
     load: u32,
     buckets: u32,
-) -> ResultBench<(String, TableSchema, Vec<T>)> {
-    let filter_column_type = match T::product_type().elements[column_index as usize].algebraic_type {
-        AlgebraicType::String => "string",
-        AlgebraicType::U32 => "u32",
-        AlgebraicType::U64 => "u64",
```
[the remark this answers is not in the document]
Comment (kazimuth) [reply crates/bench/benches/generic.rs:299]: Oh, no, I screwed up the revert, damn. Will fix.

**Comment by kazimuth** [comment]
This should be ready for merge assuming https://github.com/clockworklabs/SpacetimeDB/actions/runs/6473567337/job/17576553107 succeeds, and the tests pass...

#403 and #402 should be fixed, #401 I still have no idea about.

**Comment by kazimuth** [comment]
Ok, now this is ready for merge. I still haven't fixed the recompilation issue (#401) but I'll leave that for another day.

**Review by RReverser — APPROVED** [review]
I'm going to just go ahead and merge this as I want to unblock benchmarks on branches / PRs.

**Comment by RReverser** [comment]
Ah probably because there's pre-existing cache without those newly declared files. Might be worth cleaning cache manually.

**Comment by RReverser** [comment]
Went through caches and deleted all those from master; let's see if that helps...

**Comment by RReverser** [comment]
Nope somehow still same error even when it clearly says "cache not found": https://github.com/clockworklabs/SpacetimeDB/actions/runs/6497076599/job/17645385541

**Pull request merged**